.nuspec/Microsoft.Maui.Controls.MultiTargeting.targets

```diff
@@ -1,53 +1,53 @@
 <Project>
   <ItemGroup Condition="$(TargetFramework.StartsWith('Xamarin.iOS')) != true AND $(TargetFramework.StartsWith('net6.0-ios')) != true AND $(TargetFramework.StartsWith('net6.0-maccatalyst')) != true ">
     <Compile Remove="**\**\*.iOS.cs" />
-    <None Include="**\**\*.iOS.cs" />
```

Comment:
Is there an msbuild prop with the root bin\ and obj\ folders rather than hard coding an assumption of what these are named and where they are located?

Reply:
Yes, the .NET sdk mainly uses `$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)` for exclusions with .cs files. The PR uses that elsewhere.

https://github.com/dotnet/sdk/blob/f2cc77b60cdc2a4e8b393adf7b855b478ad79671/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props#L33

For this file there is existing use of bin and obj directories so I did the same. Line 53 has: `<Compile Remove="bin\**;obj\**" />` but I can fix this up.

Reply:
If `$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)` works, I'd use that. Otherwise you could make a new property in this file, just so we don't have `bin\**;obj\**` copy-pasted so much.

Reply:
I believe you can even change the bin/obj folders using the base intermediate and base output path properties. So this is not going to help with those cases.

Reply:
We don’t ship this file, it’s just something Maui uses to build?

Reply:
Ah, yeah. I think I thought I was somewhere else...